src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -1331,6 +1331,16 @@ public function testPassExpandedToView()
         $this->assertTrue($view->vars['expanded']);
     }
 
+    public function testPassTranslateChoicesToView()
```

Comment:
please also add a test 
1. `choice_translation_domain` defaults to `translation_domain` by default
2. choice_translation_domain inherits the parent translation domain when choice_translation_domain is null